electrum/gui/qt/confirm_tx_dialog.py

```diff
@@ -619,7 +619,7 @@ def __init__(self, *, window: 'ElectrumWindow', make_tx, output_value: Union[int
             title=_("New Transaction"), # todo: adapt title for channel funding tx, swaps
             allow_preview=allow_preview)
 
-        BlockingWaitingDialog(window, _("Preparing transaction..."), self.update)
```

Comment:
> BlockingWaitingDialog was removed is transaction_dialog, where it was not particularly useful.

I have a testnet wallet where `wallet.make_unsigned_transaction()` takes 17 seconds (which is called from self.update).
That was actually the motivation I added this wrapper in https://github.com/spesmilo/electrum/pull/5833 IIRC

Reply:
why is that useful? if it takes 17 seconds to sign the transaction, then BlockingWaitingDialog is going to block the GUI as well...

Reply:
The point of BlockingWaitingDialog was to give feedback to the user about what is going on. If you remove it without a replacement, the GUI still freezes, but now without even telling the user why.

Reply:
fixed in 6635e40a2c0d89ef68c7964d205f94287448f8aa